adapters/repos/db/lsmkv/bucket.go

```diff
@@ -89,6 +89,18 @@ type Bucket struct {
 	// keeping tombstones on compaction is optional
 	keepTombstones bool
 
+	// Init and use bloom filter for getting key from bucket segments.
+	// As some buckets can be accessed only with cursor (see flat index),
+	// where bloom filter is not applicable, it can be disabled.
+	// ON by default
+	useBloomFilter bool
+
+	// Net additions keep track of number of elements stored in bucket (of type replace).
+	// As some buckets don't have to provide Count info (see flat index),
+	// tracking additions can be disabled.
+	// ON by default
+	calcNetAdditions bool
```

Comment:
How about calling it calcNetCountAdditions or calcNetTotalAdditions?

Net typically means **Network** , so this sounds like calculating the number of additions from the network or cluster or something.

Reply:
add: or maybe drop the Net?  calcAdditions sounds good

Reply:
Field holding data in `segment` struct is called `countNetAdditions`. I can change `bucket`'s field to `calcCountNetAdditions`

Reply:
That works.  How do you feel about just countAdditions?